public/components/Flyout/flyouts/components/AlertsDashboardFlyoutComponent.js

```diff
@@ -175,15 +160,8 @@ export default class AlertsDashboardFlyoutComponent extends Component {
 
   getAlerts = async () => {
```

Comment:
Not related to changes in this PR but we shouldn't call this `getAlerts` since it doesn't return anything. Maybe just updateAlertsState or something